data/src/main/java/ca/bc/gov/data/datasource/remote/model/base/specialauthority/SpecialAuthorityPayload.kt

```diff
@@ -0,0 +1,17 @@
+package ca.bc.gov.data.datasource.remote.model.base.specialauthority
+
+import java.time.Instant
+
+/**
+ * @author: Created by Rashmi Bambhania on 24,June,2022
+ */
+data class SpecialAuthorityPayload(
+    val drugName: String? = null,
+    val effectiveDate: Instant? = null,
+    val expiryDate: Instant? = null,
+    val prescriberFirstName: String? = null,
+    val prescriberLastName: String? = null,
+    val referenceNumber: String? = null,
+    val requestStatus: String? = null,
+    val requestedDate: Instant? = null
```

Comment:
Make requestedDate non-null variable as per the API specs. This field is required. Without this field timeline cannot be decided.

Reply:
If any of the record in list has null requested date then we should not show error message instead we should show all the records which don't have requested date null